packages/ai-jsx/src/core/completion.tsx

```diff
@@ -45,19 +45,17 @@ export interface FunctionDefinition {
   parameters: FunctionParameters;
 }
 
+/**
+ * This function creates a [JSON Schema](https://json-schema.org/) object to describe
+ * parameters for a {@link FunctionDefinition}.
+ * The parameters can be described either using a record of parameter names to
+ * {@link PlainFunctionParameter} objects, or using a {@link z.ZodObject} schema object.
+ *
+ * @note If using a Zod schema, the top-level schema must be an object as per OpenAI specifications.
+ */
 export function getParametersSchema(parameters: FunctionParameters) {
-  if (parameters instanceof z.Schema) {
-    const jsonSchema = zodToJsonSchema(parameters);
-    if (!('type' in jsonSchema) || jsonSchema.type !== 'object') {
-      throw new AIJSXError(
-        `Function parameters param must be an object, but was: ${JSON.stringify(jsonSchema, null, 2)}`,
-        ErrorCode.InvalidParamSchemaType,
```

Comment:
Let's remove this from the `ErrorCode` enum if it's no longer used.

Reply:
Q: should we somehow disable that error code now?
I mean: do we care if the error codes from different versions possibly map to different things?